packages/lisk-cryptography/src/index.js

```diff
@@ -12,10 +12,19 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+import * as buffer from './buffer';
 import * as convert from './convert';
 import * as encrypt from './encrypt';
 import hash from './hash';
 import * as keys from './keys';
 import * as sign from './sign';
 
-export default Object.assign({}, convert, encrypt, { hash }, keys, sign);
```

Comment:
This is a breaking change, and requires a major release. @shuse2 thoughts?

Reply:
I think we should expose buffer function, but also make alias in the old format as well.

Reply:
There should be no collisions because they've just been ripped out of `convert` directly.

Reply:
So
```
export default Object.assign(
    {},
    // ...
    buffer,
    convert: Object.assign({}, buffer, convert),
    // ...
);
```
?

Reply:
Sorry, my mistake, using `Object.assign` in this way means the export won't change. We can include this in a minor release just fine.